pkg/protofmt/reflector.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2017 Ernest Micklei
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```

Comment:
I'm not sure this is the right way to vendor private package (I exposed `nameOfVisitee()` by renaming `NameOfVisitee()`).

Reply:
where exactly is `nameOfVisitee()` defined?

Reply:
This was taken from here: https://github.com/emicklei/proto-contrib/blob/master/pkg/protofmt/reflector.go
I thought I put it somewhere but didn't 😓 

> It seems that "capitalizing the first letter" tells Go that something should be exposed and be visible from outside of the package

Yes, that I know (that's why I exposed it), but I think it's dirty although this single file vendoring doesn't hurt that much.
Perhaps, I'd better make a PR to suggest including these helper functions in the `proto` package – this `nameOfVisitee()` function is not exported as it was only used other parts in the `protofmt` package of `proto-contrib` module.

Let's leave it for now, and fix this when we find out a better solution.